ui/src/app/views/workspace-list/component.ts

```diff
@@ -101,4 +111,15 @@ export class WorkspaceListComponent implements OnInit, OnDestroy {
   addWorkspace(): void {
     this.router.navigate(['workspace/build'], {relativeTo : this.route});
   }
+
+  get twoFactorBannerEnabled() {
+    if (this.firstSignIn == null) {
+      return false;
+    }
+    // If it is three days or more after they first sign in, deactivate the banner
```

Comment:
nit: Would be better if the comment explained why: e.g. "Don't show the banner after 3 days as their account would have been disabled had they not enabled 2-factor auth."